requirements.txt

```diff
@@ -23,4 +23,5 @@ future
 openstacksdk>=0.37
 tqdm==4.36.1
 pyzmq==18.1.1
+mock
```

Comment:
Can this be placed in the test requirements instead?  It's not really useful for ordinary users

Reply:
It is actually required for disabling progress bar and multiprocessing.

Reply:
I'm sorry, I thought this was related to the mock tests